README.md

```diff
@@ -464,6 +464,9 @@ If the value is above or equal to activation-percentage, the initial SL is calcu
 
 `new_stoploss = initial-stoploss + (actual_profit_percentage - activation_percentage)`
 
+The script also supports a timeout for the stop loss. This means that it will add a configured timeout for the stop loss to be activated. When the timeout is 60 seconds for example, when the price reaches the stop loss price, a counter on 3commas will run to count down to 0.
```

Comment:
This script also supports a configurable timeout for the stoploss, which will be activated (and updated) when the stoploss is activated. For example, with a timeout of 60 seconds, 3Commas will sell the deal (market order) when after 60 seconds the current price is still below the set stoploss price. 

Misschien wat compacter, en voor mijn gevoel daardoor duidelijker?